lib/util/parse.ts

```diff
@@ -0,0 +1,10 @@
+import { logger } from '../logger';
+
+export function safeParseJson(input: string): any {
```

Comment:
why not using schema utils?
https://github.com/renovatebot/renovate/blob/304335f94bc7f1e2a0764b8d090008a41f5b0944/lib/util/schema-utils.ts#L199

Reply:
I'm not familiar with using this approach and not looking to do something new in `bun` which isn't already done in `npm`

Reply:
Refactored it back internal now